shared/naturalcrit/codeEditor/codeEditor.jsx

```diff
@@ -142,8 +154,12 @@ const CodeEditor = createClass({
 					return `\u21A4 ${text} \u21A6`;
 				}
 			},
-			gutters : ['CodeMirror-linenumbers', 'CodeMirror-foldgutter']
+			gutters           : ['CodeMirror-linenumbers', 'CodeMirror-foldgutter'],
+			showTrailingSpace : true,
```

Comment:
I like the idea of this, but I'm not a fan of the angry red squiggles that are usually on misspelled words. Is it possible to just draw the trailing whitespace as dots (or arrows for tabs)? 
![image](https://user-images.githubusercontent.com/5878534/145656160-d290c351-f566-4a46-9b63-2459db86201d.png) Like this but only on trailing?

Reply:
Ah that's a good call! I pulled the squiggles from the code mirror tutorial for something, don't remember what that was though at this point. 

Yes I like the dots idea, I'll make that change tonight!

Reply:
Updated!